src/api/endpoint/api/publish.ts

```diff
@@ -215,27 +215,44 @@ export function publishPackage(storage: IStorageHandler, config: Config, auth: I
     }
 
     try {
-      const metadata = validateMetadata(req.body, packageName);
-      // treating deprecation as updating a package
-      if (req.params._rev || isRelatedToDeprecation(req.body)) {
-        debug('updating a new version for %o', packageName);
-        // we check unpublish permissions, an update is basically remove versions
-        const remote = req.remote_user;
-        auth.allow_unpublish({ packageName }, remote, (error) => {
-          if (error) {
-            logger.error({ packageName }, `not allowed to unpublish a version for @{packageName}`);
-            return next(error);
+      storage.getPackage({
+        name: packageName,
+        uplinksLook: true,
```

Comment:
🤔 not sure if we want to lookup to upkinks on publish, this will potentially create many failures on publish (which could be terrible experience if one single package of a big monorepo fails) which should not happen, I think we don't need this.

Reply:
Yes, get local storage version is sufficient.
Change to `false`.